test/k6/src/api/notifications/orders.js

```diff
@@ -0,0 +1,32 @@
+import http from "k6/http";
+
+import * as config from "../../config.js";
+
+import * as apiHelpers from "../../apiHelpers.js";
+
+export function getById(id, token) {
+    var endpoint = config.notifications.orders_fromId(id);
+    return getByUrl(endpoint, queryParams, token);
+  }
+
+  export function getByUrl(url, token) {
+    var params = apiHelpers.buildHeaderWithBearer(token);
+    var response = http.get(url, params);
+
+    return response;
+  }
+
+  export function getBySendersReference(sendersReference, token) {
+    var endpoint =
+      config.notifications.orders_fromSendersRef(sendersReference);
+    var params = apiHelpers.buildHeaderWithBearer(token);
```

Comment:
Headers called `params`?

Reply:
thats the practice we've followed for all apis, but we could rename it to headers. params is the name used for the property in the http function we are calling.